src/actions/attendee-actions.js

```diff
@@ -518,4 +584,8 @@ const normalizeEntity = (entity) => {
     }
 
     return normalizedEntity;
-};
\ No newline at end of file
+};
+
+export const changeAttendeeListSearchTerm = (term) => (dispatch, getState) => {
```

Comment:
this method is not used ?

Reply:
this is used at https://github.com/fntechgit/summit-admin/blob/f7761db9d75c63cbed5d170cf0b2df62cb4f9128/src/pages/attendees/summit-attendees-list-page.js#L325